api/payloads/project.go

```diff
@@ -32,7 +36,7 @@ type DeleteDeployKeyRequest struct {
 
 // NewProjectResponse TODO
 type NewProjectResponse struct {
-	ID string `json:"id"`
+	Padfile *padlfile.File `json:"id"`
```

Comment:
no need to wrap this around a `NewProjectResponse`, just marshal the padlfile directly

Reply:
👍 
Removed NewProjectResponse structs and return File directly